src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const maxDiscountPeriod = 7;
+  const minDiscountPeriod = 3;
+  const maxDiscount = 50;
+  const minDiscount = 20;
+  const discount = days >= maxDiscountPeriod ? maxDiscount
+    : days >= minDiscountPeriod ? minDiscount : 0;
```

Comment:
Avoid using nested ternary operators, the readability is low.

Reply:
Use classical `if else` statement

Reply:
replaced with if-else